client/my-sites/checkout/composite-checkout/components/international-fee-notice.tsx

```diff
@@ -1,9 +1,14 @@
 import { useSelect } from '@wordpress/data';
 import { translate } from 'i18n-calypso';
 import CheckoutTermsItem from 'calypso/my-sites/checkout/composite-checkout/components/checkout-terms-item';
+import type { WpcomCheckoutStoreSelectors } from '../hooks/wpcom-store';
+import type { ManagedContactDetails } from '@automattic/wpcom-checkout';
 
 export const InternationalFeeNotice = () => {
-	const contactInfo = useSelect( ( select ) => select( 'wpcom-checkout' )?.getContactInfo() );
+	const contactInfo: ManagedContactDetails = useSelect(
```

Comment:
Same question here. This could be undefined. What's the reason for not using the inferred value?